examples/inverse_gan_author_utils.py

```diff
@@ -0,0 +1,1969 @@
+# Copyright 2018 The Defense-GAN Authors. All Rights Reserved.
```

Comment:
Should we move this file into a new directory `examples/utils/` to distinguish it as utils from the examples?